src/main/java/roboy/linguistics/Linguistics.java

```diff
@@ -46,6 +46,16 @@
         SUCCESS, FAILURE, UNDERSPECIFIED
     }
 
+    public enum UtteranceSentiment {
+	    POSITIVE(true), NEUTRAL(null), NEGATIVE(false);

```

Comment:
please stop using booleans for three state logic
remove enum parameters, Boolean toBoolean property and the constructor

complete enum should look like this:
enum UtteranceSentiment { POSITIVE, NEUTRAL, NEGATIVE;}

Reply:
Stays. All the proposal wrt. Linguistics to be resolved in undergoing patch

Reply:
ok, but if we forget to change this, I will show this to small programmer-kids so they get nightmares